### PR TITLE
Language fix - minor EN language errors

### PR DESCRIPTION
**Pull request by sarahmccarthy123**

### CODE REVIEW

File: Documentation/Troubleshooting/TYPO3.rst [hunk 1]
```diff
@@ -16,13 +16,12 @@ Resetting Passwords
 Backend Administrator Password
 ------------------------------
 
-When the password for a backend user needs to be reset, log into the backend with an
-alternative user and use the :guilabel:`System > Backend Users` tool to reset
-the users password. Note that only backend users with administrative rights can
+When the password for a backend user needs to be reset, log into the backend as a different user and use the :guilabel:`System > Backend Users` tool to reset
+the password. Note that only backend users with administrative rights can
```
Comment (mbrodala) [inline Documentation/Troubleshooting/TYPO3.rst:20]:
```suggestion
When the password for a backend user needs to be reset, log into the backend as a
different user and use the :guilabel:`System > Backend Users` tool to reset
the password. Note that only backend users with administrative rights can
```

File: Documentation/Troubleshooting/TYPO3.rst [hunk 2]
```diff
@@ -16,13 +16,12 @@ Resetting Passwords
...
 access the `Backend Users` tool to make this change.
 
 If an alternative administrator account is not available or it doesn't have the
-appropriate access, the Install Tool can be accessed directly
+appropriate access, access the Install Tool directly
```
Comment (mbrodala) [inline Documentation/Troubleshooting/TYPO3.rst:24]:
```suggestion
appropriate access, open the Install Tool directly
```

Here _access, access_ sounds weird. ;-)

File: Documentation/Troubleshooting/TYPO3.rst [hunk 3]
```diff
@@ -33,7 +32,7 @@ when TYPO3 was installed.
 .. include:: ../Images/AutomaticScreenshots/InstallTool/InstallToolPassword.rst.txt
 
 Once logged in to the Admin Tool go to :guilabel:`Maintenance > Create Administrative User`
-and select :guilabel:`Create Administrator`. In this dialogue you
+and select :guilabel:`Create Administrator`. In this dialog box you
```
Comment (mbrodala) [inline Documentation/Troubleshooting/TYPO3.rst:35]:
```suggestion
and select :guilabel:`Create Administrator`. Here you
```

File: Documentation/Troubleshooting/TYPO3.rst [hunk 4]
```diff
@@ -86,14 +85,14 @@ Debug Settings
...
-change to show errors in the frontend.
+changed to show errors in the frontend.
 
 .. include:: ../Images/AutomaticScreenshots/DebugSettings/ConfigurationPresets.rst.txt
 
 .. include:: ../Images/AutomaticScreenshots/DebugSettings/DebugSettings.rst.txt
 
-The following TypoScript setting can also be added the the root template for
-the site to show additional debug information.
+The following TypoScript setting can also be added into the root template for
```
Comment (mbrodala) [inline Documentation/Troubleshooting/TYPO3.rst:94]:
```suggestion
The following TypoScript setting can also be added to the root template for
```

File: Documentation/Troubleshooting/TYPO3.rst [hunk 5]
```diff
@@ -106,8 +105,8 @@ This is particularly useful when debugging Fluid errors:
 
 .. important::
 
-   Once debugging has been completed, make sure to remove any debug Typoscript and
-   set Debug setting back to 'Live'.
+   Once debugging has been completed, make sure you remove any debug Typoscript and
```
Comment (mbrodala) [inline Documentation/Troubleshooting/TYPO3.rst:108]:
```suggestion
   Once debugging has been completed, remove any debug TypoScript and
```

File: Documentation/Troubleshooting/TYPO3.rst [hunk 6]
```diff
@@ -145,7 +144,7 @@ you will find::
 
 These files contain all :file:`ext\_tables.php` and
 :file:`ext\_localconf.php` files of the installed extensions
-concatenated in the order they are loaded. Therefore including one of
+concatenated in the order that they are loaded. Therefore including one of
```
Comment (mbrodala) [inline Documentation/Troubleshooting/TYPO3.rst:147]:
```suggestion
concatenated in their loading order. Therefore including one of
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Comment by flicstar** [comment]
Thanks heaps @sarahmccarthy123 for your work on this! I have reviewed and I think if we can incorporate @mbrodala's suggested changes, we're good to go 👍

**Review by linawolf — APPROVED** [review]

**Review by sarahmccarthy123 — COMMENTED** [review]
Yes this all looks fine :-)

**Review by mbrodala — APPROVED** [review]

**Pull request merged**